### DISCUSSION TITLE
Comparison of African country statistics by ruggedness index

### DISCUSSION THREAD

**Discussion by t-pappas** [Chris-Nicholas-collaborators/EEP153_team_Malthus#5] [Ideas]
This was an indicator shown to be correlated with increase/decrease in slave trade and has had lasting effects on GDP

**Comment by chrisgnicholas** [comment]
Do you mean "ruggedness" according to topography?

This is an interesting paper -
https://pseweb.eu/ydepot/semin/texte0708/PUG2008RUD.pdf

This is an interesting GIS problem..
I can generate this kind of thing based using topography ... 
https://docs.qgis.org/3.4/en/docs/user_manual/processing_algs/qgis/rasterterrainanalysis.html#ruggedness-index

and and then correlate  population statistics on a per-grid cell basis, 
https://hub.worldpop.org/geodata/summary?id=16289 
and these are HUGE image files - ~400 megs per column

It would be basically a demographic curve for every pixel ... 
now, not sure how this would aggregate to compare with other nation-wide statistics ...

We'd also want to look at access, which could be done relative to the road network.

But ... not sure if we want to attempt this in 2 weeks ... ?
